UM/Application.py

```diff
@@ -101,6 +101,12 @@ def __init__(self, name: str, version: str, build_type: str = "", **kwargs):
 
         self._operation_stack = OperationStack(self.getController())
 
+        # Ensure configuration folder has a folder for themes
+        try:
+            os.makedirs(os.path.join(Resources.getStoragePath(Resources.Resources), "themes"))
```

Comment:
This should not be needed as Resources.getStoragePath() already does makedirs() if the directory does not exist.

See https://github.com/Ultimaker/Uranium/blob/master/UM/Resources.py#L167